aspnetcore/publishing/iis.md

```diff
@@ -401,11 +412,11 @@ Troubleshooting:
 
 Troubleshooting:
 
-* Confirm that you have enabled the proper role. See [IIS Configuration](#iis-configuration).
+* Confirm that you've enabled the proper role. See [IIS Configuration](#iis-configuration).
 
-* Check **Programs &amp; Features** and confirm that the **Microsoft ASP.NET Core Module** has been installed. If the **Microsoft ASP.NET Core Module** is not present in the list of installed programs, install the module. See [Install the .NET Core Windows Server Hosting bundle](#install-the-net-core-windows-server-hosting-bundle).
+* Check **Programs &amp; Features** and confirm that the **Microsoft ASP.NET Core Module** has been installed. If the **Microsoft ASP.NET Core Module** isn't present in the list of installed programs, install the module. See [Install the .NET Core Windows Server Hosting bundle](#install-the-net-core-windows-server-hosting-bundle).
 
-* Make sure that the **Application Pool > Process Model > Identity** is set to **ApplicationPoolIdentity** or your custom identity has the correct permissions to access the application's deployment folder.
+* Make sure that the **Application Pool > Process Model > Identity** is set to **ApplicationPoolIdentity** or your custom identity has the correct permissions to access the app's deployment folder.
```

Comment:
Please bold each individual item in the hierarchy of steps:
**Application Pool** > **Process Model** > **Identity**